daemon/algod/api/server/v2/handlers.go

```diff
@@ -732,6 +732,81 @@ func (v2 *Handlers) GetBlockTxids(ctx echo.Context, round uint64) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
+func NewBlockLog(txid string, logs []string, appIndex uint64) *model.BlockLog {
```

Comment:
You can return `model.BlockLog` here. All call sites dereference and throw it away anyway.